Automattic/jetpack
# VideoPress: Fix video caption saving

Fixes #26441.

#### Changes proposed in this Pull Request:

* Allow the `caption` as a possible field on the video meta update payload
* Set the attachment `post_excerpt` field with the submited `caption` value, as per pe4Cmx-fL-p2#comment-153
* Set the caption on the attachment meta information, as we do for the other fields on all non-wpcom sites

#### Other information:


#### Jetpack product discussion



N/A.

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* Test on a site with VideoPress installed
* Go to `Media > Library` and upload a new video
* Go to `Jetpack > VideoPress` to see your VideoPress library
* Click the `Edit video details` button on the video that you just uploaded
* Fill the Caption field with some value and click `Save changes`
* Click the `Edit video details` button again
* The Caption field must be filled with the same text that you typed before
* Do a hard refresh on the page and check the Caption field value again: it must have the same typed value
